### DISCUSSION TITLE
Dịch Generative Learning

### DISCUSSION THREAD

**Discussion by hnguyentt** [mlbvn/handson-ml2-vn#160] [Terminology]
Mình thấy trong glossary có dịch các từ: generative adversarial network (mạng đối sinh), generative model (mô hình sinh), generator (bộ sinh).
Mình nên dịch `Generative Learning` là gì cho hợp lý nhỉ? Chắc không phải là `Học Sinh` =))

**Comment by quangnhat185** [comment]
`Học Khởi sinh` thì sao nhỉ?

**Reply by minhduc0711** [reply]
Mình thấy "khởi sinh" có nghĩa tựa như "bắt đầu" hơn. Mình thấy có mẫy chỗ dịch là "tạo sinh".

**Comment by hnguyentt** [comment]
Mình đang tạm dịch là "học sinh tạo"

**Reply by lkhphuc** [reply]
học tạo sinh ? 🧐

**Reply by ngcthuong** [reply]
Học sinh tạo nghe nó hơi khó cắt từ. 
Mình nghe ra thành "học sinh" tạo:). Mình thấy học tạo sinh dễ nghe hơn.
